rules/windows/c2_rundll32_sequence.toml

```diff
@@ -22,16 +22,20 @@ tags = ["Elastic", "Windows"]
 type = "eql"
 
 query = '''
+/* replace winlog.event_data.OriginalFileName with process.pe.original_file_name once field is available in winlogbeat */
+
 sequence by process.entity_id with maxspan=2h
   [process where event.type in ("start", "process_started") and
-     (process.name == "rundll32.exe" or process.pe.original_file_name == "rundll32.exe") and
+     (process.name == "rundll32.exe" or process.pe.original_file_name == "rundll32.exe" or
+        winlog.event_data.OriginalFileName == "rundll32.exe") and
```

Comment:
I think from an offline discussion we mentioned that we should avoid non-ECS fields as much as possible.
The reason for this, is because EQL isn't as forgiving as KQL when it comes to unknown fields. In KQL, it'll accept whatever you give it. But in EQL, if you try to search only `logs-endpoint.*` with this rule, it will fail.